src/agl/WebAppManagerServiceAGL.h

```diff
@@ -3,20 +3,42 @@
 
 #include <memory>
 
+#include "WebRuntimeAGL.h"
 #include "WebAppManagerService.h"
 #include "Timer.h"
+#include "AglShellSurface.h"
 
 constexpr char kStartApp[] = "start-app";
 constexpr char kKilledApp[] = "killed-app";
 constexpr char kActivateEvent[] = "activate-event";
 constexpr char kDeactivateEvent[] = "deactivate-event";
+constexpr char kSendAglReady[] = "ready-event";
 
 class WamSocket;
 class WamSocketLockFile;
 
 class WebAppManagerServiceAGL : public WebAppManagerService {
 public:
     static WebAppManagerServiceAGL* instance();
+    class StartupArgs {
+    public:
+       StartupArgs(std::string _app_id, std::string _app_uri,
```

Comment:
Use const & for all elements we don't want to copy once more when calling this constructor. Also, at least for now, we follow Qt convention, so private properties would be prefixed with m_ (and the constructor parameters would not use any prefix or suffix).

Be careful with the extra copies of std::list.

Reply:
Okay, const & them, and used m_ notation.